run.py

```diff
@@ -6,16 +6,12 @@
 
 root = Path(__file__).resolve().parent
 
-simulator_name = VUnit.from_argv(compile_builtins=False).get_simulator_name()
-vhdl_standard = "2019" if simulator_name in ["rivierapro", "activehdl"] else "2008"
-print(f"Using VHDL-{vhdl_standard}")
-
-cli = VUnitCLI()
-args = cli.parse_args()
-args.keep_compiling = True
```

Comment:
This was actually deliberate. I was thinking that the users that will download this and run with their own simulator offline should have to use a minimal amount of options. Have no strong opinion on this though so I will accept either way.

Reply:
As a middle ground, I added a comment with a refence to http://vunit.github.io/py/ui.html#adding-custom-command-line-arguments. See https://github.com/VHDL/Compliance-Tests/pull/15/files#diff-d6af0459a37d985953d7040c14f53feb3b9cc9e58b543aa3c2b80256d276c5e0R8-R10.